eo-phi-normalizer/src/Language/EO/Phi/Rules/Yaml.hs

```diff
@@ -172,6 +173,7 @@ data Subst = Subst
   { objectMetas :: [(MetaId, Object)]
   , bindingsMetas :: [(MetaId, [Binding])]
   , attributeMetas :: [(MetaId, Attribute)]
+  , functionMetas :: [(MetaFunctionName, Object)]
```

Comment:
As an alternative, I could just reuse `objectMetas`, either using `MetaId` instead of `MetaFunctionName`, or relaxing the key here to just `String`.

Reply:
Removed entirely.